org.ekstep.questionunit-1.0/renderer/plugin.js

```diff
@@ -1,6 +1,16 @@
 org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
   _type: 'org.ekstep.questionUnitPlugin',
   _render: true,
+  _addUrduSupport: function(){
+    // Add to Question Title
+    var urduSupportClass = 'urdu-right-align'
+    $('.question-title-text-common-questionunit-plugins').addClass(urduSupportClass);
+
+    // Add to Question Options
+    _.each($('.option-text-common-questionunit-plugins'), function(option){
+      $(option).addClass(urduSupportClass);
```

Comment:
Verify for null/undefined check here before applying addClass.
$(option)

Reply:
Resolved as part of 
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/